crates/iota-core/src/authority.rs

```diff
@@ -2220,10 +2231,14 @@ impl AuthorityState {
         Ok(epoch_store.reference_gas_price())
     }
 
-    pub fn is_tx_already_executed(&self, digest: &TransactionDigest) -> IotaResult<bool> {
+    pub fn try_is_tx_already_executed(&self, digest: &TransactionDigest) -> IotaResult<bool> {
         self.get_transaction_cache_reader()
             .try_is_tx_already_executed(digest)
     }
+    pub fn is_tx_already_executed(&self, digest: &TransactionDigest) -> bool {
```

Comment:
Add a blank line here 😁 
```suggestion
    }
    
    pub fn is_tx_already_executed(&self, digest: &TransactionDigest) -> bool {
```